_studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp

```diff
@@ -218,28 +214,22 @@ mfxStatus VideoDECODEH265::Init(mfxVideoParam *par)
     m_vPar.mfx.NumThread = (mfxU16)CalculateNumThread(par, m_platform);
 
     if (MFX_PLATFORM_SOFTWARE == m_platform)
```

Comment:
MFX_CHECK

Reply:
@dmitry-gurulev comment, please

Reply:
Checked CS code base. Would be hard to align if we will use macro here